### PR TITLE
feat: added request to add specific tiles to queue

### PR DESCRIPTION
**Pull request by CptSchnitz**

### CODE REVIEW

File: src/tiles/routes/tilesRouter.ts [hunk 1]
```diff
@@ -7,6 +7,7 @@ const tilesRouterFactory: FactoryFunction<Router> = (dependencyContainer) => {
   const controller = dependencyContainer.resolve(TilesController);
```
Comment (melancholiai) [inline src/tiles/routes/tilesRouter.ts:7]: export both the router and the symbol in the same fashion

File: src/tiles/models/tilesManager.ts [hunk 2]
```diff
@@ -48,6 +48,11 @@ export class TilesManager {
     }
   }
 
+  public async addTilesToQueue(tiles: Tile[]): Promise<void> {
+    const tileArr = tiles.map((tile) => ({ name: this.tilesQueueName, data: tile }));
```
Comment (melancholiai) [inline src/tiles/models/tilesManager.ts]: rename to something that indicates that it is in fact jobs

File: tests/integration/tiles/tiles.spec.ts [hunk 3]
```diff
@@ -46,80 +46,140 @@ describe('tiles', function () {
     });
 
     describe('Happy Path', function () {
-      it('should return ok', async function () {
-        const bbox = getBbox();
+      describe('POST /tiles/bbox', function () {
+        it('should return ok', async function () {
```
Comment (melancholiai) [inline tests/integration/tiles/tiles.spec.ts]: be a little more generous

File: tests/integration/tiles/tiles.spec.ts [hunk 4]
```diff
@@ -46,80 +46,140 @@ describe('tiles', function () {
...
+          const response = await requestSender.postTilesList([{ z: 0, x: 0, y: 0, metatile: 8 }]);
+
+          expect(response.status).toBe(httpStatusCodes.OK);
+
+          expect(response).toSatisfyApiSpec();
+        });
       });
     });
     describe('Bad Path', function () {
```
Comment (melancholiai) [inline tests/integration/tiles/tiles.spec.ts:71]: test every different case where an error is invoked by `validateTile`

File: tests/integration/tiles/tiles.spec.ts [hunk 5]
```diff
@@ -46,80 +46,140 @@ describe('tiles', function () {
...
+        it('should return 400 if the body contains tile that is out of bounds', async function () {
+          const response = await requestSender.postTilesList([{ z: 0, x: 1, y: 1, metatile: 8 } as Tile]);
+
+          expect(response.status).toBe(httpStatusCodes.BAD_REQUEST);
+          expect(response.body).toHaveProperty('message', 'x index out of range of tile grid');
+          expect(response).toSatisfyApiSpec();
+        });
       });
     });
     describe('Sad Path', function () {
```
Comment (melancholiai) [inline tests/integration/tiles/tiles.spec.ts:164]: add new line between happy and bad and bad and sad

**Review by melancholiai — CHANGES_REQUESTED** [review]

File: src/tiles/models/tilesManager.ts (hunk 2 above)
Comment (CptSchnitz) [reply src/tiles/models/tilesManager.ts:52]: its not

File: src/tiles/routes/tilesRouter.ts (hunk 1 above)
Comment (CptSchnitz) [reply src/tiles/routes/tilesRouter.ts:7]: fixed

File: src/tiles/models/tilesManager.ts (hunk 2 above)
Comment (CptSchnitz) [reply src/tiles/models/tilesManager.ts]: renamed

File: tests/integration/tiles/tiles.spec.ts (hunk 4 above)
Comment (CptSchnitz) [reply tests/integration/tiles/tiles.spec.ts:71]: testing every case is wrong, they are already covered by validateTile tests.

File: tests/integration/tiles/tiles.spec.ts (hunk 3 above)
Comment (CptSchnitz) [reply tests/integration/tiles/tiles.spec.ts]: added more text

File: tests/integration/tiles/tiles.spec.ts (hunk 5 above)
Comment (CptSchnitz) [reply tests/integration/tiles/tiles.spec.ts:164]: done

**Review by melancholiai — APPROVED** [review]
run prettier

**Pull request merged**